app/js/lib/foxbox/sequential-timer.js

```diff
@@ -0,0 +1,94 @@
+const p = Object.freeze({
+  isStarted: Symbol('isStarted'),
+  nextTickHandle: Symbol('nextTickHandle'),
+
+  // Private methods.
+  scheduleTick: Symbol('scheduleTick'),
+  onTick: Symbol('onTick'),
+});
+
+export default class SequentialTimer {
+  /**
+   * Creates new SequentialTimer instance.
+   * @param {number} interval Minimum interval between two consequent ticks.
+   */
+  constructor(interval) {
+    this.interval = interval;
+
+    this[p.isStarted] = false;
+    this[p.nextTickHandle] = null;
+    this[p.onTick] = null;
+  }
+
+  /**
+   * Indicates whether timer started or not.
+   *
+   * @return {boolean}
+   */
+  get isStarted() {
+    return this[p.isStarted];
```

Comment:
I would have called it `hasStarted`, but maybe `started` is good enough: quick and easy to remember.